docs/configuration/configuration.md

```diff
@@ -2199,6 +2199,45 @@ The following meta labels are available on targets during [relabeling](#relabel_
 # The information to access the Nomad API. It is to be defined
 # as the Nomad documentation requires.
 [ server: <host> | default = "http://localhost:4646" ]
+
```

Comment:
tag_separator, region and allow_stale are not documented

Reply:
added those just now, i also added the more generic parameters below i hope thats ok/not sure i was meant to do that